contrib/pkg_tester/.gitignore

```diff
@@ -0,0 +1 @@
+.venv
```

Comment:
Do we need this file here? Shouldn't this be added to the main `.gitignore`?

Reply:
Done